Week1/Section3/Exercise9/Source9.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Exercise 9 - String Reversal
+ * We�ve seen that strings are just arrays of characters. Create a char array and initialize it with any message.
+ * The char array should have a size of 30 independently of the size of your string.
+ * Output your string and reverse it using a loop and a second char array, then output the reversed string.�
+ * Do not hard code the size of your string when looping through it.
+ */
+#include <iostream>
+
+int main()
+{
+	char MyArray[42] = "_Seeking vengeance against the dark lords";
+
+	std::cout << MyArray << std::endl;
+
+	const int ArrayLength = (sizeof(MyArray) / sizeof(char));
+	int AuxiliarLength = ArrayLength - 1;
+
+	char ArrayVolteao[ArrayLength]; // creates a second array with the size of the first
```

Comment:
Our programs can be funny and who doesn't love a fun program, but that humor stays in the program, not in our code. Our code should respect the standards and prioritize legibility and organization. With that in mind, `ArrayVolteao` is not an acceptable name here.

Reply:
ok, variable name changed